src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy

```diff
@@ -104,13 +104,45 @@ class FlowHistoryV2Spec extends HealthCheckSpecification {
         }
 
         when: "Update the created flow"
-        flowHelperV2.updateFlow(flow.flowId, flow.tap { it.description = it.description + "updated" })
+        def oldPathCompStrategy = PathComputationStrategy.LATENCY.toString()
+        def newPathCompStrategy = PathComputationStrategy.COST.toString()
+        flowHelperV2.updateFlow(flow.flowId, flow.tap {
```

Comment:
same. You can easily do it if instead of updating original `flow` you update a copy of it.
`def updatedFlow = flow.jacksonCopy()`

Reply:
thx, fixed